addon/components/infinity-loader.js

```diff
@@ -38,14 +39,32 @@ const InfinityLoaderComponent = Ember.Component.extend({
     this.get('_scrollable').off(`${eventName}.${this.get('guid')}`);
   },
 
-  _checkIfInView() {
-    var selfOffset       = this.$().offset().top;
-    var scrollable       = this.get("_scrollable");
-    var scrollableBottom = scrollable.height() + scrollable.scrollTop();
+  _selfOffset() {
+    if(this.get('_customScrollableIsDefined')) {
```

Comment:
use `if (` for consistency - a single space between keywords and open parens, to distinguish from functions, a la Crockford